dashboard/modules/dashboard_sdk.py

```diff
@@ -348,6 +348,15 @@ def _upload_package(
                     include_parent_dir=include_parent_dir,
                     excludes=excludes,
                 )
+
+            package_size = package_file.stat().st_size
+            if package_size > 100 * 1024 * 1024:  # 100 MiB
```

Comment:
Maybe there is a constant somewhere in the ray library that we can reuse that represents the maximum upload? While your comment makes it clear what the ["magic numbers"](https://en.wikipedia.org/wiki/Magic_number_(programming)) are, it would be clearer if there were a constant. Additionally, if the Ray devs change the maximum upload size in the future, it would make sure that this code remains future-proof.

Unfortunately I'm not a ray dev, so I'm not sure where they set or configure the limit (or if it's even accessible in python) so I'll leave that to ray devs to decide